packages/compute-baseline/src/baseline/support.ts

```diff
@@ -13,6 +13,18 @@ export function support(feature: Feature, browsers: Browser[]): Support {
   const support: Support = new Map();
   for (const b of browsers) {
     const releases = feature.supportedBy({ only: [b] });
+    // TODO:
+    // let lastInitial: Release | undefined;
+    // let lastInitialBoundary: "" | "≤" | undefined;
+
+    // const reverseChronological = b.releases.slice().reverse();
+    // let previousRelease: string | undefined;
+    // for (let index = 0; index < reverseChronological.length; index++) {
+    //   const release = reverseChronological[index] as Release;
+    //   const current = feature.flatSupportedIn(release);
+
+    //   // Check if current has changed, etc.
+    // }
```

Comment:
This is just to show where I'm going with this: walk backward through releases, then find the support-to-non-support boundary. If we stepped into a `null`, then that's `≤X`.